api/agentlocalpb/agentlocal.proto

```diff
@@ -21,13 +21,37 @@ option (grpc.gateway.protoc_gen_swagger.options.openapiv2_swagger) = {
 option go_package = "agentlocalpb";
 
 import "google/api/annotations.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+import "inventory/agent_status.proto";
 import "protoc-gen-swagger/options/annotations.proto";
 
-message StatusRequest {}
+// ServerInfo returns information about server which agent was connected.
+message ServerInfo {
+  string url = 1;
+  bool insecure_tls = 2;
+  google.protobuf.Timestamp last_ping_time = 3;
+  google.protobuf.Duration latency = 4;
+}
+
+// AgentInfo returns information about agent which was runned by pmm-agent.
+message AgentInfo {
```

Comment:
Add agent type

Reply:
What is agent_type? We didn't talk about it. :/

Reply:
You know what agent type is, and it is already known to pmm-agent